src/scripts/modal.js

```diff
@@ -1,15 +1,19 @@
 function modal() {
   const heroContent = document.querySelector(".hero__content");
+  const projectContent = document.querySelector(".projects");
```

Comment:
Not the best idea. Whenever we add a section you would add it's element to the modal script?

Reply:
Any ideas?